pkg/kotsadm/types/constants.go

```diff
@@ -13,7 +13,7 @@ const ExcludeValue = "true"
 const BackupLabel = "kots.io/backup"
 const BackupLabelValue = "velero"
 
-const TroubleshootKey = "troubleshoot.io/kind"
+const TroubleshootKey = "troubleshoot.sh/kind"
```

Comment:
As per https://github.com/replicatedhq/troubleshoot/pull/1075#issuecomment-1482656575, we need to maintain both labels and instead do the following

* When users upgrade to a newer KOTS version, update secrets and configmaps that have `"troubleshoot.io/kind"` label to `"troubleshoot.sh/kind"` label.
* Maintain searching using both labels for sometime. Use cases such as searching for labels from objects restored from a backup need to be supported.